src/abstract_trees.jl

```diff
@@ -28,6 +28,10 @@ apart from the two points mentioned.
 In analogy to the type definitions of `DecisionTree`, the generic type `S` is 
 the type of the feature values used within a node as a threshold for the splits
 between its children and `T` is the type of the classes given (these might be ids or labels).
+
+Please note: You may only add lacking class labels. It's not possible to overwrite  
+existing labels with this mechanism. In case you want add class labels, 
+the generic type `T` must be a subtype of `Integer`.
```

Comment:
```suggestion
    !!! note
    You may only add lacking class labels. It's not possible to overwrite existing labels
    with this mechanism. In case you want add class labels, the generic type `T` must 
    be a subtype of `Integer`.
```
It's a Julia convention to use such admonitons (https://documenter.juliadocs.org/stable/showcase/#Note-admonition)